source/manual/rummager-traffic-replay.html.md

```diff
@@ -0,0 +1,35 @@
+---
+owner_slack: "#search-team"
+title: 'Replay HTTP traffic on rummager machines'
+parent: "/manual.html"
+layout: manual_layout
+section: Backups
+last_reviewed_on: 2017-09-05
+review_in: 3 months
+---
+
+Have have started using [GOR](https://github.com/buger/goreplay) to store POST and DELETED requests
+to the rummager servers.
```

Comment:
This should probably say "requests made to the rummager servers" as it sounds a bit like the rummager servers are just the place where you're storing stuff.

Reply:
Done